### ISSUE TITLE
Create a Custom Component for Popup Dialogs

### ISSUE THREAD

**Issue by erenfn** [bluewave-labs/guidefox#434]
Create a custom component for popup dialogs and change existing popup dialog designs (like the one in #367, the one below, delete account popup in settings page etc.)

The paddings, font sizes, shadows, border-radii etc. will be same but the component will get Header, Content, Left button text, and Right button text as props.
 
![image](https://github.com/user-attachments/assets/a59f82d1-95af-4502-a833-2844f8b2ab20)

**Comment by Yup03** [comment]
hi, i saw that this issue is not assigned,
Is there someone woking on it???
If not, can you add me to the assignees list so i can try to solve this??

**Comment by erenfn** [comment]
Sure, you can take this one

**Comment by erenfn** [comment]
Hey, how is this task going?

**Comment by Yup03** [comment]
Please excuse the long waiting. I'm a little stuck on the the configuration part so far, 
It's bit of a hassle running the project on my windows machine
Is there a way that I can create the component without the project maybe running locally on my computer???

**Comment by erenfn** [comment]
Configuration is for hosting the application on a server. You can just clone the project and run docker compose up

**Comment by Yup03** [comment]
Thanks, I was not aware of that. Been trying to configure this whole time and didn't get any work around
Gonna try it and will come back to give feedbacks

**Comment by Yup03** [comment]
I was able to take a look at the project
The way that I understand your comment is that a new component should be created, but I spotted a component called **_PopUpMessages_** in the components folder and that one doesn't show up anywhere in the UI
And my question is: 
Is it the **_PopUpMessages_** component that should be really modified or should I really create a brand new component ???
Any additional information would be welcomed.

**Comment by erenfn** [comment]
Yes, PopupMessages component should be used. But the variables like Header, Content, Left button text, and Right button text, type should be props, so it needs some changes
